cin_validator/utils.py

```diff
@@ -1,6 +1,8 @@
 from copy import deepcopy
 
+import numpy as np
 import pandas as pd
+from govuk_bank_holidays.bank_holidays import BankHolidays
```

Comment:
wow this is really cool! Who maintains this? I'm so glad it exists!

Reply:
Same here. 
I was quite happy to see it. It is maintained by the ministry of Justice https://github.com/ministryofjustice/govuk-bank-holidays

This is the pip page https://pypi.org/project/govuk-bank-holidays/